acceptance/write_proxy_test.go

```diff
@@ -0,0 +1,177 @@
+package influxtest
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+)
+
+func (s *Suite) Test_WriteLineProtocol_SingleMetric() {
+	line := fmt.Sprintf("stat,unit=temperature,status=measured avg=%f", 23.5)
+	err := s.api.writeAPI.WriteRecord(context.Background(), line)
+	s.Require().NoError(err)
+
+	expectedResult := model.Sample{
+		Metric: model.Metric{
+			model.MetricNameLabel:               "stat_avg",
+			model.LabelName("__proxy_source__"): "influx",
+			model.LabelName("unit"):             "temperature",
+			model.LabelName("status"):           "measured",
+		},
+		Value: 23.5,
+	}
+
+	result, _, err := s.api.querierClient.Query(context.Background(), "stat_avg", time.Now())
```

Comment:
can you add a query before the record is written to confirm that the result before and after the write is different?

Reply:
correct